src/supermin-init-prelude.sh

```diff
@@ -25,11 +28,19 @@ fi
 mkdir -p "${workdir}"/cache /host/container-work
 mount /dev/sdb1 "${workdir}"/cache
 
+if [ -f "${workdir}/tmp/supermin/supermin.env" ]; then
+    source "${workdir}/tmp/supermin/supermin.env";
```

Comment:
i tend to think that this is a file that gets created and destroyed upon each invocation where it is needed since multiple things run `runvm` - should we do the cleanup (`rm`) here to guarantee that?

Reply:
I'm going to push back on that: if you are defining run-time parameters in your environment, its on you to ensure its okay.  If we clean it up, then some steps like `coreos/cmd-oscontainer` will lose this later.

Reply:
right - so it depends on what it is. For example. If I set these env vars intended for `cmd-oscontainer` but then I run `cmd-build` does it hurt things? If the answer is they are global and are fine for all calls then that is fine. I just tended to think of it as a these variables are for this operation so we'd need to set/unset them for each operation.

Reply:
It should not hurt anything, unless you're doing some whonky things. In testing, this ran through multiple steps with out breaking anything; the testing was not limited to one step. And since this is in the root of `tmp`, `cmd-build` uses its own `tmp`, eg `tmp/build/tmp`. So its safe.